LondonBritishLibrary/orient/BLorient810.xml

```diff
@@ -30,20 +31,88 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 385</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="incomplete" ref="LIT1011Abusak"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>Chapters 47, 49, 51, 53, 58 and 59 are missing.</note>
+
+                          <msItem xml:id="ms_i1.1">
+                            <locus from="4r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Index"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                            <locus from="7r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter1"></title>
+                            <textLang mainLang="gez"></textLang>
```

Comment:
From the description it is now not clear where are chapters 2-48, it looks like a mistake. Can their presence be indicated as ms-items without just loci?